Sources/Phakchi/ControlServer.swift

```diff
@@ -1,25 +1,36 @@
 import Foundation
 
 public class ControlServer {
-    public static let defaultServer = ControlServer()
+    public static let `default` = ControlServer()
```

Comment:
Changing the name might be a good idea. I'm not a big fan of using keywords as method names.